packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -155,7 +156,7 @@ Content-Type: text/json;charset=UTF-8
 
 {
     "schemas": ["urn:ietf:params:scim:api:messages:2.0:Error"],
-    "detail": "Conflict - User already exists",
+    "detail": "User already exists in the database.",
```

Comment:
In Line 148 above, I was confused by the formatting, since we use that `>` format for notes, rather than UI messages. Maybe regular text, within quotation marks, would work instead?

Reply:
OK - have moved to a "" surrounded version